gradle-plugin/src/main/java/com/squareup/anvil/plugin/AnvilExtension.kt

```diff
@@ -16,15 +18,39 @@ open class AnvilExtension {
    *
    * By default this feature is disabled.
    */
-  var generateDaggerFactories = false
+  abstract val generateDaggerFactoriesProperty: Property<Boolean>
+
+  var generateDaggerFactories: Boolean
```

Comment:
We _could_ use `@JvmName` to make this a source breaking change only if we want, but this approach keeps this non-breaking in terms of both source and binary